src/App.tsx

```diff
@@ -1,25 +1,53 @@
+import { useState, useEffect } from "react";
 import "./App.css";
 import About from "./components/organisms/About/About";
+import Footer from "./components/organisms/Footer/Footer";
 import Gallery from "./components/organisms/Gallery/Gallery";
-import structure from "./structure.json";
 
 const App = () => {
-  const about = structure.about;
-  const gallery = structure.gallery;
+  const [data, setData] = useState<any>();
```

Comment:
'any' type for useState is not appropriate. try to come up with your own data type

Reply:
I've came up with solution, hope this will work, if not will proceed with changes again